docs/identity/authentication/tutorial-enable-sspr.md

```diff
@@ -96,7 +96,7 @@ An administrator can manually provide this contact information, or users can go
 1. To apply the registration settings, select **Save**.
 
 > [!NOTE]
-> The interruption to request to register contact information during signing in only occurs if the conditions configured on the settings are met. This only applies to users and admin accounts that are enabled to reset passwords using Microsoft Entra self-service password reset. 
+> The request to register contact information during signing in only occurs if the conditions configured on the settings are met. This only applies to users and admin accounts that are enabled to reset passwords using Microsoft Entra self-service password reset. 
```

Comment:
Great catch! Let's say "The interruption to register..." because this is sometimes known as "interrupt mode" for admins who configure authentication method registration